demisto_sdk/commands/common/content/objects/abstract_objects/dictionary_based_object.py

```diff
@@ -63,5 +63,12 @@ def get(self, key: str, default: Optional[Any] = None) -> Any:
 
         return value
 
+    def get_id(self):
+        as_dict = self.to_dict()
+        if 'commonfields' in as_dict.keys():
+            return as_dict.get('commonfields', {}).get('id')
+        else:
+            return as_dict.get('id')
```

Comment:
use `get_entity_id_by_entity_type`

Reply:
As discussed, remaining as is since the 'type' used has value mismatch (Integrations vs. Integration) and the object's attribute is only of an inheriting object.